applications/nrf_desktop/doc/nrf_profiler_sync.rst

```diff
@@ -0,0 +1,50 @@
+.. _nrf_desktop_nrf_profiler_sync:
+
+Profiler synchronization module
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Use the nrf_profiler synchronization module to synchronize the timestamps of :ref:`nrf_profiler` events between two devices connected over a physical wire.
+Timestamp synchronization is required to increase the accuracy of the measured times between the nrf_profiler events that come from two different devices.
+
+The nrf_profiler data is collected separately from both devices using dedicated Python scripts.
+The data can then be merged using the :file:`merge_data.py` script with ``sync_event`` used as a synchronization event for both ``Peripheral`` and ``Central``.
+For more details, see the :ref:`nrf_profiler` documentation.
+
+Module events
+*************
+
+.. include:: event_propagation.rst
+    :start-after: table_nrf_profiler_sync_start
+    :end-before: table_nrf_profiler_sync_end
+
+.. note::
+    |nrf_desktop_module_event_note|
+
+Configuration
+*************
+
+A predefined signal on the GPIO is used to simultaneously generate synchronization nrf_profiler events on both devices.
+For this reason, you must enable the :kconfig:option:`CONFIG_GPIO` option.
+
+You must also enable the :kconfig:option:`CONFIG_APP_EVENT_MANAGER_NRF_PROFILER_TRACER` Kconfig option.
+The nrf_profiler synchronization module generates a :ref:`nrf_profiler` event (``sync_event``) that is not an :ref:`app_event_manager` event.
+For this reason, the ``sync_event`` execution is not traced.
+
+You must also define the following:
+
+* The GPIO port (:ref:`CONFIG_DESKTOP_NRF_PROFILER_SYNC_GPIO_PORT <config_desktop_app_options>`) and the pin (:ref:`CONFIG_DESKTOP_NRF_PROFILER_SYNC_GPIO_PIN <config_desktop_app_options>`) that are used for synchronization.
+  These GPIOs must be defined separately for both devices and connected using a physical wire.
+* The device role.
+  One of the devices must be set as ``Central`` (:ref:`CONFIG_DESKTOP_NRF_PROFILER_SYNC_CENTRAL <config_desktop_app_options>`) and the other device must be set as ``Peripheral`` (:ref:`CONFIG_DESKTOP_NRF_PROFILER_SYNC_PERIPHERAL <config_desktop_app_options>`).
+
+Implementation details
+**********************
+
+The nrf_profiler synchronization ``Central`` generates a predefined signal that is received by ``Peripheral`` over the physical wire.
```

Comment:
```suggestion
The nrf_profiler's synchronization Central generates a predefined signal that is received by Peripheral over the physical wire.
```
Can we use terms "synchronization central" and "synchronization peripheral" throughout the doc, maybe?